apps/aeutils/priv/aeternity_config_schema.json

```diff
@@ -923,7 +923,133 @@
                     "default" : "ae_mainnet"
                 }
             }
+        },
+        "stratum": {
+            "type"    : "object",
+            "additionalProperties": false,
+            "properties" : {
+                "enabled" : {
+                    "type" : "boolean",
+                    "default" : false
+                },
+                "connection": {
+                    "type"    : "object",
+                    "additionalProperties": false,
+                    "properties" : {
+                        "host" : {
+                            "type" : "string",
+                            "default" : "pool.aeternity.com"
```

Comment:
Shall this be amended to something more generic?

Reply:
definitely we need to remove "aeternity.com"

Reply:
I've put in `localhost`.